IFC4x3/Properties/n/NominalDiameter_0LC_x6u$f4sgZiVYKmGCvh/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalDiameter_0LC_x6uf4sgZiVYKmGCvh" Name="NominalDiameter" UniqueId="1533eec6-e3fa-44da-a8ec-7e253040ce6b" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Should this be a harmonized property?

Reply:
Done. Harmonized with Generic Nominal Diameter definition. Commit : 305806220fb93c9010687f137214b3f48df310cb